charts/kong/example-values/README.md

```diff
@@ -23,6 +23,20 @@ change this value to `true` if you use Helm 2).
   installs Kong for Kubernetes with Kong Enterprise with the ingress controller
   in PostgreSQL. It enables all Enterprise services.
 
+* [minimal-kong-hybrid-control.yaml](minimal-kong-hybrid-control.yaml) and
+  [minimal-kong-hybrid-data.yaml](minimal-kong-hybrid-data.yaml) install
+  separate releases for hybrid mode control and data plane nodes, using the
+  built-in PostgreSQL chart on the control plane release. They require some
+  pre-work to [create certificates](https://github.com/Kong/charts/blob/master/charts/kong/README.md#certificates)
+  and configure the control plane location. See comments in the file headers
+  for additional details.
+
+  Note that you should install the control plane release before the data plane
+  release if possible: data planes must be able to talk with a control plane
+  node before they can come online. Data plane nodes will retry this for a time
+  before evicted by liveness checks, however, so starting them second isn't
+  strictly required so long as the control nodes come online shortly after.
```

Comment:
maybe possible to write with fewer words?

```suggestion
  Note that data plane pods will crash-loop with liveness checks until they can 
  access the control plane. Therefore it's recommended (but not strictly necessary) 
  to start the control plane before the data plane.
```

Reply:
Made a different change to instead keep roughly the same length but remove as much K8S-specific terminology as possible--ideally users do know what liveness checks and backoff are, but many are less experienced and don't.